lib/pencilblue-cli.js

```diff
@@ -1,64 +1,61 @@
 #! /usr/bin/env node
 
-/*
- * pencilblue-cli
- * https://pencilblue.org
- *
- * Copyright (c) 2014 PencilBlue
- * Licensed under the MIT license.
- */
-
-'use strict';
+var colors = require('colors/safe');
+var process = require('process');
 
-global.fs = require('fs');
-global.shell = require('shelljs');
-global.prompt = require('prompt');
-global.moduleVersion = "0.2.0";
+var info = require('./services/info');
+var install = require('./services/install');
 
-//configure environment
-var process = require('process');
 process.on('uncaughtException', function(err) {
-    console.error(err);
-    process.exit(1);
+  console.log(colors.red('Error: ' + err));
+  process.exit(1);
 });
 process.on('SIGINT', function() {
   process.exit(0);
 });
 
-var InstallService = require('./InstallService');
-var instService = new InstallService();
-
-var InfoService = require('./InfoService');
-var infoService = new InfoService();
-
-var LaunchService = require('./LaunchService');
-var launchService = new LaunchService();
-
-var args = process.argv;
+var argv = process.argv;
 
-if(!args[2]) {
-    infoService.getHelp(args[1]);
+if(typeof argv[2] === 'undefined') {
+  info.getHelp(true);
 }
 else {
-    switch(args[2]) {
-        case 'help':
-        case '--help':
-        case '-h':
-            infoService.getHelp(args[1]);
-            break;
-        case 'version':
-        case '--version':
-        case '-v':
-            infoService.getVersion(args[1]);
-            break;
-        case 'install':
-            instService.install(args[3] || 'pencilblue');
-            break;
-        case 'start':
-            launchService.startPB(args[3]);
-            break;
-        default:
-            infoService.getHelp(args[1]);
-            break;
-    }
+  switch(argv[2]) {
+    case 'version':
+    case '--version':
+    case '-v':
+      info.getVersion().then(function(version) {
+        console.log(colors.white(version));
+      }, function(err) {
+        console.log(colors.red(err));
+      });
+      break;
+    case 'install':
+      var version = null;
+      var directory = 'pencilblue';
+      if(typeof argv[4] !== 'undefined') {
+        version = argv[3];
+        directory = argv[4];
+      }
+      else if(typeof argv[3] !== 'undefined'){
+        directory = argv[3];
+      }
+      install.installWithSettings(version, directory).then(function() {
+        console.log(colors.green('Installation completed.'));
+      }, function(err) {
+        throw(err);
+      });
+      break;
+    case 'start':
+      console.log(colors.yellow('The start command is deprecated. Use `npm start` instead.'));
+      break;
+    case 'help':
+    case '--help':
+    case '-h':
+      info.getHelp(true);
+      break;
+    default:
+      info.getHelp(true);
+      break;
```

Comment:
it is redundant to separate the default case and the last ```'-h'``` case because they do the same thing.  You can modify it to the below and reduce the LOC.  Additionally, you don't need the last ```break;``` statement because you will fall out of the switch anyway.
```
case '--help':
case '-h':
default:
  info.getHelp(true);
}
```